audformat/core/database.py

```diff
@@ -289,6 +290,47 @@ def drop_tables(
         for table_id in table_ids:
             self.tables.pop(table_id)
 
+    def files_duration(
+            self,
+            files: typing.Union[str, typing.Sequence[str]],
+    ) -> typing.Union[pd.Timedelta, pd.Series]:
+        r"""Duration of file(s) in the database.
+
+        Note that durations are cached,
+        i.e. changing the files on disk after calling
+        this function can lead to wrong results.
+        The cache is cleared when the
+        database is reloaded from disk.
+
+        Use ``db.files_duration(db.files).sum()``
+        to get the total duration of a database.
+        Or ``db.files_duration(db['table'].files).sum()``
```

Comment:
I would change it to
```suggestion
        Or ``db.files_duration(db[table_id].files).sum()``
```

Reply:
Changed it accordingly.